mmv1/products/privateca/api.yaml

```diff
@@ -845,11 +1182,358 @@ objects:
+                      - !ruby/object:Api::Type::Boolean
+                        name: 'timeStamping'
+                        description: |
+                          Corresponds to OID 1.3.6.1.5.5.7.3.8. Officially described as "Binding the hash of an object to a time".
+                      - !ruby/object:Api::Type::Boolean
+                        name: 'ocspSigning'
+                        description: |
+                          Corresponds to OID 1.3.6.1.5.5.7.3.9. Officially described as "Signing OCSP responses".
+                  - !ruby/object:Api::Type::Array
+                    name: 'unknownExtendedKeyUsages'
+                    description: |
+                      An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+                    item_type: !ruby/object:Api::Type::NestedObject
+                      properties:
+                        - !ruby/object:Api::Type::Array
+                          name: 'objectIdPath'
+                          required: true
+                          item_type: Api::Type::Integer
+                          description: |
+                            An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'publishingOptions'
+        description: |
+          The PublishingOptions to follow when issuing Certificates from any CertificateAuthority in this CaPool.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'publishCaCert'
+            required: true
+            description: |
+              When true, publishes each CertificateAuthority's CA certificate and includes its URL in the "Authority Information Access"
+              X.509 extension in all issued Certificates. If this is false, the CA certificate will not be published and the corresponding
+              X.509 extension will not be written in issued certificates.
+          - !ruby/object:Api::Type::Boolean
+            name: 'publishCrl'
+            required: true
+            description: |
+              When true, publishes each CertificateAuthority's CRL and includes its URL in the "CRL Distribution Points" X.509 extension
+              in all issued Certificates. If this is false, CRLs will not be published and the corresponding X.509 extension will not
+              be written in issued certificates. CRLs will expire 7 days from their creation. However, we will rebuild daily. CRLs are
+              also rebuilt shortly after a certificate is revoked.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: labels
+        description: |
+          Labels with user-defined metadata.
+
+          An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass":
+          "1.3kg", "count": "3" }.
+
+
+
```

Comment:
is it possible to add a new resource for CertificateTemplates?

Reply:
Definitely possible in the future. We decided to omit this from the initial implementation